source/vocab/relations.ttl

```diff
@@ -392,6 +393,7 @@ prefix relsubtype: <http://id.loc.gov/vocabulary/preservation/relationshipSubTyp
     owl:inverseOf :replacedBy .
 
 :hasReproduction a owl:ObjectProperty;
+    :category :integral ; # TODO: is this really the case, or is :reproductionOf enough?
```

Comment:
According to the comment on integral (nicely documented by the way 😃 ) it, I would not say a reproduction is necessery to understand the original resource.

Reply:
Probably not (and thanks 😊 ). What I had in mind was the perceived need to show the media of a reproduction alongside a print; but that is not *integral* information (there are lots of interesting things we do show which are not). OTOH it *might* be the case that `:associatedMedia` is `:integral` though, but we'll see.

I'll just remove the `:integral` from `:hasReproduction` for now and we'll see if any such need arises.

Reply:
Removed in  3fd9af2.